src/utils/ChoicesWrapper.js

```diff
@@ -68,7 +68,6 @@ class ChoicesWrapper extends Choices {
       return;
     }
 
-    this.shouldOpenDropDown = false;
```

Comment:
This completely negates the need for this variable. For code cleanup, can we just remove it all other places it is referenced.